packages/core/src/checkout-buttons/checkout-button-options.ts

```diff
@@ -19,6 +19,7 @@ export interface CheckoutButtonOptions extends RequestOptions {
 }
 
 export interface CheckoutButtonInitializeOptions extends CheckoutButtonOptions {
+    [ key: string ]: unknown;
```

Comment:
I don't think you need this after the rebase.

Reply:
I rebased and this was still needed for the types to match :|